### ISSUE TITLE
Remove `h5` element from byline markup.

### ISSUE THREAD

**Issue by benlk** [INN/Largo#1787]
Having headings out of order messes with the document outline, according to various tests.

> To solve this violation, you need to:
> Fix the following:
> 
> Heading order invalid

- [ ] replace `h5` wrapper markup in various partials with a ... `span`, I guess.
- [ ] replace `h5.byline` styles with `.byline` across the board, and check visual appearances

---- 
This issue was identified during research on https://github.com/INN/largo/issues/1653 using the AXE browser plugin.

This ticket replaces https://github.com/INN/largo/issues/1139.
